pymove/utils/_winmock.py

```diff
@@ -0,0 +1,16 @@
+class WindowsUser:
+  """Mock class for module errors on windows."""
+  def __init__(self):
+    """Instantiates mock class"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 's') (D415)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)